### DISCUSSION TITLE
Langchain using typescript

### DISCUSSION THREAD

**Discussion by sidharth-14** [langchain-ai/langserve#284] [Q&A] [answered]
I want to create chain using typescript and use those chains to make the api endpoint using langserve. Is this possible and are there in documentation I can refer to.

**Comment by eyurtsev** [comment, marked as answer]
Hi @sidharth-14, 

At the moment, the langserve backend is entirely in python, so you'll need to write the chain in python as well.

Langchain js and python are very similar to though, so it may not be too difficult to port over your changes to python.

Could you say a bit more about your use case?
